<launch>  
    <node name="hand_tracking" type="handtracker_webcam_RGB.py" pkg="hand_tracking" >
        <!-- <remap from="/usb_cam/image_raw" to="/camera/color/image_raw"/>     -->
        <remap from="/usb_cam/image_raw" to="/spencer/sensors/rgbd_front_top/color/image_raw"/>    

    </node>
</launch>